Firestore/core/test/unit/bundle/bundle_serializer_test.cc

```diff
@@ -31,21 +31,35 @@ using nlohmann::json;
 using ProtoBundleMetadata = ::firestore::BundleMetadata;
 using util::ReadContext;
 
-TEST(BundleSerializerTest, DecodesBundleMetadata) {
-  auto proto_metadata = ProtoBundleMetadata{};
+ProtoBundleMetadata bundle_metadata() {
```

Comment:
Nit: please rename to use `PascalCase` (`snake_case` is only for class getters and setters).

Reply:
Done.